### PR TITLE
[11.0][IMP] sale_margin_delivered: take an account returned moves

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Why not using directly the quantity received shown in sale.order.line? It takes into account returned quantity when you mark the "To refund" check. With this, I think there will be inconsistent data.

**Comment by sergio-teruel** [comment]
¿Quantity received?

**Comment by pedrobaeza** [comment]
Sorry, "quantity delivered" is the term (I think the field is `qty_delivered`).

**Comment by sergio-teruel** [comment]
Ok, i understand, I the customer return me goods but I do not check to refund the delivered qty is the initial delivered quantity.... and in this case i would discard this stock moves....

**Review by yajo — APPROVED** [review]

File: account_invoice_margin/models/account_invoice.py [hunk 1]
```diff
@@ -53,12 +53,6 @@ def _compute_margin(self):
     def _onchange_product_id_account_invoice_margin(self):
         if self.invoice_id.type in ['out_invoice', 'out_refund']:
             purchase_price = self.product_id.standard_price
-            if any(self.product_id.supplier_taxes_id.mapped('price_include')):
```
Comment (pedrobaeza) [inline account_invoice_margin/models/account_invoice.py:56]: Are you sure this is not needed?

Comment (sergio-teruel) [reply account_invoice_margin/models/account_invoice.py:56]: Yes!! Also is incorrect do that!!

**Pull request merged**